README.md

```diff
@@ -91,7 +91,7 @@ metadata:
 type: Opaque
 fieldRef:
 
-  # So, <key> is NOT a placeholder 
+  # So, <key> is NOT a placeholder
   description: 'Selects a field of
     the pod: supports metadata.name,
     metadata.namespace, `metadata.labels[''<KEY>'']`,
```

Comment:
Not relevant to this PR at all but can you change `<key>` to `<KEY>` so the example is more clear. I got the casing wrong